iso.bbx

```diff
@@ -250,8 +327,11 @@
      \printnames{supervisor}}%
 }
 
-%titles
+% TITLES MACROS
 
+% Macro for formatting <prefix>titles
+% 1st argument: prefix / type of title (e.g. main, book, event)
```

Comment:
It might be wort mentioning that the prefix/type may be empty, in which case the macro will handle the `title` field.